src/index.css

```diff
@@ -0,0 +1,446 @@
+*{
```

Comment:
Your CSS is laid out very nicely in the same layout as the HTML which makes it easy to follow, nice job!